File: CarDealerWebAPI/Core.CarDealer/Models/Image.cs
using System;
using System.Collections.Generic;

namespace Core.CarDealer.Models
{
    public partial class Image
    {
        public Guid Id { get; set; }
        public string ImageName { get; set; } = null!;
        public Guid? CarId { get; set; }

        public virtual Car? Car { get; set; }
    }
}

File: CarDealerWebAPI/CarDealer.Client/ClientApp/src/app/Models/Image.ts




export class Image
{
    private _id!: string;
    public get Id(): string 
    {
        return this._id;
    }
    public set Id(value: string) 
    {
        this._id = value;
    }
    private _imageName!: string;
    public get imageName(): string
    {
        return this._imageName;
    }
    public set imageName(value: string) 
    {
        this._imageName = value;
    }
    private _carId!: number;
    public get carId(): number 
    {
        return this._carId;
    }
    public set carId(value: number) 
    {
        this._carId = value;
    }
}